src/twisted/internet/iocpreactor/tcp.py

```diff
@@ -560,6 +560,11 @@ def handleAccept(self, rc, evt):
             )
             return False
         else:
+            # Inherit the properties from the listening port socket as
+            # documented in the `Remarks` section of AcceptEx.
+            # https://msdn.microsoft.com/en-us/library/ms737524(v=VS.85).aspx
```

Comment:
While that URL still works, it now redirect to:
https://docs.microsoft.com/en-us/windows/win32/api/mswsock/nf-mswsock-acceptex